common/views/themes/config.ts

```diff
@@ -136,10 +136,10 @@ export const themeValues = {
     medium: 600,
     large: 960,
     xlarge: 1338,
-  },
-  tweakpoints: {
+    // Tweakpoints
```

Comment:
I think the reason I wanted to avoid this was that we do some `Object.entries` mapping on the `sizes` key to create various `utility-classes`. I wanted to keep the ones that already existed, but didn't want to add any more, because quite a lot of extra css would be created. I don't know if this would be reflected in the performance action we run? I guess if it doesn't create too much extra stuff, then it's ok – but maybe doesn't feel particularly great?

Reply:
The main issue is I couldn't see any places where we don't merge the tweakpoints and breakpoints, so the nuance between them isn't being used anywhere.

This was part of why I did this as I was just about to merge them to allow me to use [utility classes](https://github.com/wellcomecollection/wellcomecollection.org/blob/main/common/views/themes/default.ts#L123-L146). Do you have an example of where we don't want to do this and it might be an issue?  

The perf would be monitored in the page size.

Reply:
I think it looks like we may have removed the instances where we were looping through all the sizes, so perhaps this isn't the possible problem I thought it could be. Are we reporting on the actual page sizes (rather than the js bundles)?

Reply:
Good shout, we're not - which will be hard as content is really dynamic, but we should probably look at reporting on CSS.